draft-ietf-modpod-group-processes.md

```diff
@@ -238,15 +249,22 @@ Administrators are primarily responsible for managing their fora in
 accordance with guidance developed by the moderators and approved by
 the IESG. As such, they shall address reports of inappropriate behavior
 in a timely fashion, apprising moderators of their disposition.
+Administrators should take seriously the need for consistent
+implementation of moderation guidelines.
 
 Moderators are responsible for establishing processes to
 address moderation needs across all IETF fora, both present and
 future.  Moderators are expected to be a resource that the community
 can use to address problematic contributions.
 
 Moderators may take actions when administrators do not respond to
-reports in a timely fashion.  Their first action should generally be to
-attempt to contact the relevant administrators.
+reports in a timely fashion.  Their first action should generally be
+to attempt to contact and advise the relevant administratorsl.
+They should only take
+moderation actions when administrators are not responsive.  In
+particular moderators should give WG chairs every opportunity to
```

Comment:
nit: should be "In particular," [with comma]

Reply:
There are two general rules for leading prepositional phases:

- More than one and you must separate with commas.  Example: On a dark night, in a cold house, David lit a fire.
- A single but long phrase.  In the middle of a war in the Pacific, Harry and Ito got drunk on a bottle of saki.

For short single phrases, the comma is optional.

Reply:
I this one needs the comma, because the difference between "In particular moderators" and "In particular, moderators" does matter. It resolves itself if you keep going, but it doesn't read well imho. I tried a few grammar checkers, and they agree.

Reply:
Right, it's not a big deal. The way I think about it is reading from a teleprompter when you don't know what is coming next:

- "In particular moderators, we find a high level of empathy"
- "In particular, moderators should..."